src/pages/home/report/ReportActionsView.js

```diff
@@ -453,13 +423,10 @@ export default compose(
     withDrawerState,
     withLocalize,
     withNetwork(),
+
     withOnyx({
-        isLoadingReportData: {
-            key: ONYXKEYS.IS_LOADING_REPORT_DATA,
-        },
-        isLoadingMoreReportActions: {
-            key: ({reportID}) => `${ONYXKEYS.COLLECTION.IS_LOADING_MORE_REPORT_ACTIONS}${reportID}`,
-            initWithStoredValues: false,
+        report: {
+            key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
```

Comment:
We are already passing the report as a prop to this component so we don't need `withOnyx` anymore

https://github.com/Expensify/App/blob/972b83549e10cf1f05310f269f07a2a83b1e2253/src/pages/home/ReportScreen.js#L242-L245